specification/workloadmonitor/resource-manager/microsoft.workloadmonitor/preview/2018-08-31-preview/Microsoft.WorkloadMonitor.json

```diff
@@ -620,14 +617,84 @@
             }
           }
         },
-        "x-ms-pageable": {
-          "nextLinkName": "nextLink"
-        },
         "x-ms-examples": {
           "NotificationSettings_ListByResource": {
             "$ref": "./examples/NotificationSettings_ListByResource.json"
           }
         }
+      },
```

Comment:
@vivek-microsoft - The PUT needs a {name} in the URL for the notificationSettings. If its a singleton, we can use "default" as the name. Do you intend to allow creation of multiple notificationSettings through this API? If yes, then thats not currently supported.